events/cognito.go

```diff
@@ -1,21 +1,57 @@
-// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 
 package events
 
-// CognitoEvent contains data from an event sent from AWS Cognito
-type CognitoEvent struct {
```

Comment:
CognitoEvent should not be removed. It is the model used for the Cognito Sync integration- https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-events.html

PreSignup and PostConfirmation, referenced by #44, are a part of Cognito User Pools. https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-lambda-trigger-examples.html#aws-lambda-triggers-post-confirmation-example